cheroot/test/ssl/ca.cert

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I'd like to see those ssl stubs generated (and cleaned up) by pytest fixtures rather then be part of the repo.

Reply:
(this way it would be like self-documentation for using this feature as well)